src/test/java/teammates/test/cases/browsertests/InstructorSearchPageUiTest.java

```diff
@@ -109,7 +111,31 @@ private void testSearch() throws Exception {
         searchPage.verifyHtmlMainContent("/instructorSearchPageSearchStudentsForStudent2.html");
     }
 
+    private void testDeleteAction() throws Exception {
+
+        ______TS("action: delete");
+
+        searchPage.clearSearchBox();
+        String searchContent = "\"student2 2 In Course1\"";
+        searchPage.inputSearchContent(searchContent);
+        searchPage.clickSearchButton();
```

Comment:
Is it possible to reuse the previous search result for testing of deletion? This will result in an additional page load, and add to the testing time.

Reply:
@whipermr5 I believe it is possible, but this will cause this delete test to be coupled and depend on the previous test...which I am not sure is such a good idea :/

Reply:
@whipermr5 Should I still go ahead with attempting to reuse the test result?